haskell/private/versions.bzl

```diff
@@ -0,0 +1,34 @@
+# check_version below cannot be called from anywhere
+# because native.bazel_version's availability is restricted:
+#   https://github.com/bazelbuild/bazel/issues/8305
+# An alternative hacky way is as follows:
+#   https://github.com/protocolbuffers/upb/blob/master/bazel/repository_defs.bzl
+#
+# There are utility functions for parsing versions numbers here:
+#   load("@bazel_skylib//lib:versions.bzl", "versions")
+# But we don't want to use them, as skylib is not yet loaded when code
+# in this file executes (and there's no way to execute it later; see first
+# paragraph above).
+
+def check_version(actual_version):
+    if type(actual_version) != "string" or len(actual_version) < 5:
+        return  # Unexpected format
+
+    expected_version = "2.0.0"  # Change THIS LINE when changing bazel version
```

Comment:
In `start` we define a range of valid Bazel versions. Why restrict it to a single version here, rather than using the same range?

Reply:
Yes indeed, at the time I wrote this code it was unclear which version we're supported; and then we had the Bazel meeting (last week) and I made the release which made things clearer. I will update the PR accordingly.

Reply:
I used the same range in `versions.bzl` and `start` in fccd1ae824c5a0e6a3747edde5cc34fec25390a9.